aurielle-auriellep/hw_2/5-fizzbuzz/app.js

```diff
@@ -0,0 +1,30 @@
+// Relying on your newfound knowledge of loops, combine loops and if/else statements together and incrementally build the common fizzbuzz loop.
+
+// - In the loop, every time a number is divisible by **3**, instead of logging the number itself, the word "fizz" should appear.
+// - If the number is divisible by  **5**, the word "buzz" should be logged.
+// - If the number is divisible by both **3** and  **5**, then the word "fizzbuzz" should be logged.
+
+// Follow the steps below.
+
+// ##### Step 1:
+// Construct a for loop that iterates through, and `console.log()`'s out, numbers 1 - 100.
+
+
+const divideMe = (num) => {
+    while (num >= 2) {
+        num--;
+        let byThree = num % 3;
+        let byFive = num % 5;
+        let byBoth = (byThree === 0 && byFive === 0) ? 'fizzbuzz' : undefined;
```

Comment:
Error handling _and_ a ternary operator? Wowza! If ternary operators are most comfortable to you, I encourage you to keep using it; just watch for line length (<80 characters is generally considered best practice).